simpeg/electromagnetics/base_1d.py

```diff
@@ -237,16 +237,43 @@ def compute_complex_sigma(self, frequencies):
         r"""
         Computes the complex conductivity matrix using Pelton's Cole-Cole model:
 
+        Parameters
+        -----------
+        frequencies : array_like
+            (n_frequency,)-array containing frequencies
+
+            Warning
+            -------
+            For IP cases (non-null chargeability), computations assume
+            conductivity/frequency at infinite frequency were passed (see notes
+            below)
```

Comment:
Instead of using a section title here (flake complains about it), we can turn it into an admonition:
```suggestion
            .. important::

                For IP cases (non-null chargeability), computations assume
                conductivity/frequency at infinite frequency were passed
                (see notes below).
```